include/boost/gil/image_processing/fast_feature_detector.hpp

```diff
@@ -0,0 +1,170 @@
+#ifndef BOOST_GIL_IMAGE_PROCESSING_FAST_HPP
+#define BOOST_GIL_IMAGE_PROCESSING_FAST_HPP
+
+#include <boost/gil/image.hpp>
+#include <boost/gil/image_view.hpp>
+#include <boost/gil/locator.hpp>
+#include <boost/gil/point.hpp>
+#include <algorithm>
+#include <cmath>
+#include <vector>
+namespace boost { namespace gil {
+namespace detail {
+template <typename srcview>
```

Comment:
- blank line missing
- `srcview` is template parameter - template parameters must read CamelCase so `SrcView`

Reply:
Inserted Blank lines, Camelcase used